back/db/migrate/20220207103216_create_projects_topics.rb

```diff
@@ -0,0 +1,10 @@
+class CreateProjectsTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :projects_topics, id: :uuid do |t|
+      t.uuid :project_id, null: false, foreign_key: true, index: true
```

Comment:
According to `schema.rb`, foreign key is not created. I believe you should use `references` method from the link I've sent before.